website/content/docs/concepts/scheduling/placement.mdx

```diff
@@ -109,6 +109,20 @@ facilitate managing the relationships between jobs, namespaces, and node pools.
 
 Refer to the [Node Pools][concept_np] concept page for more information.
 
+## Understanding Evaluation Status
+
+When a job cannot be immediately placed, Nomad creates a chain of evaluations
+to manage placement. First, your initial evaluation may be marked complete even if 
+placement fails, then a blocked evaluation is created to retry placement when
+resources become available. Your job will remain pending till all allocations
+are placed.
```

Comment:
```suggestion
placement fails. Then Nomad creates a blocked evaluation to retry placement when
resources become available. Your job remains pending until Nomad places all allocations.
```

Suggestion to match our docs style guide. 
- We use [active voice](https://developers.google.com/style/voice) so I changed "is created" to "Nomad creates" and "all allocations
are placed" to "Nomad places all allocations."
- We use present tense so I changed "will remain" to "remains"